Network/Network.cpp

```diff
@@ -240,9 +240,11 @@ namespace WPEFramework
                     {
                         JsonObject interface;
                         std::string iface = m_netUtils.getInterfaceDescription(list.interfaces[i].name);
+#ifdef NET_DEFINED_INTERFACES_ONLY
```

Comment:
Probably not needed, as an event with an empty "interface" parameter would not really be useful to anybody.